cmd/aem/content.go

```diff
@@ -32,25 +33,31 @@ func (c *CLI) contentCleanCmd() *cobra.Command {
 				c.Error(err)
 				return
 			}
+			file, err := determineContentFile(cmd)
+			if err != nil {
+				c.Error(err)
+				return
+			}
 			if dir != "" {
 				if err = c.aem.ContentManager().CleanDir(dir); err != nil {
 					c.Error(err)
 					return
 				}
-			}
-			file, _ := cmd.Flags().GetString("file")
-			if file != "" {
+				c.SetOutput("dir", dir)
```

Comment:
one of the assumptions of AEMC are deterministic outputs; in that way we will have different JSON/YML structure dependant on the input parameters; I mean we should always output `path` or both fields